src/ixdat/techniques/cv.py

```diff
@@ -324,6 +325,15 @@ def diff_with(self, other, v_list=None, cls=None, v_scan_res=0.001, res_points=1
         return diff
 
 
+class CyclicVoltammagram(CyclicVoltammogram):
+
+    # FIXME: decorating the class itself doesn't work because the callable returned
```

Comment:
Wow. Had not considered that. Nice catch.

Reply:
The reason I was angling to avoid this, was that I was unsure if the actual class name was being used anywhere in (de)/serialization code and so would cause a data hygiene problem. If that is the case, we might just move ahead and hard deprecate immediately.

Reply:
I will raise an issue on it, since I've documented being able to use it for the class.